server/src/test-data.js

```diff
@@ -51,11 +56,13 @@ export const loadTestData = () =>
               group,
               name: 'Wollongong S&W',
               details: 'Weekly storm availability',
+              startTime: 0, // ongoing
```

Comment:
ok didn't realise you wanted to support this, sure. You probably want to store an ongoing endTime as some constant timestamp set far into the future though so that DB querying/indexes work.